app/routers/experiment_templates.py

```diff
@@ -26,6 +30,62 @@ async def get_experiment_templates(pagination: Pagination = Depends()) -> Any:
     ]
 
 
+@router.get(
+    "/experiment-templates/approved", response_model=list[ExperimentTemplateResponse]
+)
+async def get_approved_templates(pagination: Pagination = Depends()) -> Any:
+    experiment_templates = await ExperimentTemplate.find(
+        ExperimentTemplate.approved == TRUE,
+        skip=pagination.offset,
+        limit=pagination.limit,
+    ).to_list()
+
+    return [
+        experiment_template.map_to_response()
+        for experiment_template in experiment_templates
+    ]
+
+
+@router.get("/experiment-templates/my", response_model=list[ExperimentTemplateResponse])
+async def get_my_experiment_templates(
+    user: Json = Depends(get_current_user), pagination: Pagination = Depends()
+) -> Any:
+    experiment_templates = await ExperimentTemplate.find(
+        ExperimentTemplate.created_by == user["email"],
+        skip=pagination.offset,
+        limit=pagination.limit,
+    ).to_list()
+
+    return [
+        experiment_template.map_to_response()
+        for experiment_template in experiment_templates
+    ]
+
+
+@router.get(
+    "/experiment-templates/all-view", response_model=list[ExperimentTemplateResponse]
```

Comment:
It is getting too complicated with this endpoint. We can maybe change it to be de default `GET /experiment-templates`

Reply:
Well, it's up to debate. Because I wanted to keep the original `/experiment-templates` functionality the same so that it gives you all the existing templates in database, regardless their attributes.

On the other hand the `/experiment-templates/all-view` variant returns those templates that are being shown on frontend. Actually the original `/experiment-templates` route is not being used as of now, but if we do change it the way 
youre recommending, then it might be a bit misleading as the renamed `/experiment-templates` route wouldnt return all templates.

What do you think?